src/core/public/i18n/i18n_service.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { i18n } from '@kbn/i18n';
+import { InjectedMetadataStartContract } from '../injected_metadata';
+interface Deps {
+  injectedMetadata: InjectedMetadataStartContract;
+}
+export class I18nService {
+  public start({ injectedMetadata }: Deps) {
```

Comment:
This method isn't actually a start method, but I named as `start` to be consistent with other services.

Reply:
Maybe at some point we'll do more than just `i18n.init`, so I'm +1 for consistency.